configure-alertmanager-operator-services/configure-alertmanager-operator.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
+  name: configure-alertmanager-operator
+  path: /hack/olm-registry/olm-artifacts-template.yaml
```

Comment:
this path doesn't exist yet, right?

Reply:
Is that in https://github.com/openshift/configure-alertmanager-operator ?

Reply:
It is, working on a PR.

Reply:
@jmelis https://github.com/openshift/configure-alertmanager-operator/pull/16 was merged, can you have another look here?